CMake/resolve_dependency_modules/boost/CMakeLists.txt

```diff
@@ -65,8 +65,5 @@ list(REMOVE_ITEM BOOST_INCLUDE_LIBRARIES headers)
 set(BUILD_SHARED_LIBS ON)
 FetchContent_MakeAvailable(Boost)
 
-# To aling with Boost system install we create Boost::headers target
-add_library(boost_headers INTERFACE)
-add_library(Boost::headers ALIAS boost_headers)
```

Comment:
Why are we removing these?

Reply:
Because the newer boost version already has these targets defined. If you leave these in then you get a cmake error about duplicate targets.

Reply:
Which boost version introduced this?
Can you add this detail to the PR description as well? Thanks.

Reply:
Looks like version 1.82 introduced them: https://github.com/boostorg/headers/commit/1060751a7897ed9f9756666a5fcf348b8ceb4006